packages/test-utils/types/index.d.ts

```diff
@@ -69,6 +69,7 @@ interface BaseWrapper {
 
   trigger (eventName: string, options?: object): void
   destroy (): void
+  selector: Selector | void
```

Comment:
This isn't actually a Selector type, it's an object with a value and type: https://github.com/vuejs/vue-test-utils/blob/dev/packages/test-utils/src/get-selector.js#L38

Because the library isn't written in TS, we have tests (https://github.com/vuejs/vue-test-utils/blob/dev/packages/test-utils/types/test/wrapper.ts) that access and set the wrapper options using the types. Can you write a test that accesses selector.value and selector.type and update this types file.

Reply:
> This isn't actually a Selector type

@eddyerburgh Thank you for catching this! :+1:

It actually is of type Selector for the ErrorWrapper: https://github.com/vuejs/vue-test-utils/blob/70ac83a95324841ea9839228bf9eb2ab6603a3ad/packages/test-utils/src/error-wrapper.js#L22

I will update the other places to use `rawSelector` (which is of type Selector), too, because that prevents us from defining a new type and consumers of the API don't need to do `selector.value` to get the selector they provided in the first place.

> we have tests (https://github.com/vuejs/vue-test-utils/blob/dev/packages/test-utils/types/test/wrapper.ts)

Sure, I can add tests there! :+1: 

Right now I'm a little confused by the structure of that file though because it has neither test cases in the usual style (no `it('something happens', () => { ... })` / `test(...)` blocks) nor assertions (only implicit ones). Could you provide me with an example of how such a test would look like?

Reply:
> It actually is of type Selector for the ErrorWrapper: 

Ah, it turns out this was wrong actually—but the Selector type is currently `any`:

https://github.com/vuejs/vue-test-utils/blob/70ac83a95324841ea9839228bf9eb2ab6603a3ad/flow/wrapper.flow.js#L6

I will adjust the places that create ErrorWrapper instances to use the rawSelector as well then.

Reply:
Sure, the test script just compiles the file to make sure there aren't any compiler errors when using the Vue Test Utils API, caused by incorrect typing.

A test would look something like this:

```js
let s: string = wrapper.selector.value
```

Reply:
Thanks! :bowing_man: I'll add the tests and ping you again once they are ready.